src/main/java/com/github/xetorthio/jedisque/Jedisque.java

```diff
@@ -17,6 +13,9 @@
 	public Jedisque() {
 		super();
 	}
+	public  Jedisque(int timeout){
```

Comment:
What If I want to set URIs and a timeout? I can't set it with this constructor. We should provide another constructor that allows URIs and a timeout together.